services/QuillLMS/spec/controllers/cms/rosters_controller_spec.rb

```diff
@@ -87,6 +88,53 @@
       expect(JSON.parse(response.body)).to be_empty
     end
 
+    it 'should overwrite password for existing student if the parameter overwritePasswords is true' do
+      create(:schools_users, school: school, user: existing_teacher)
+      new_password = "new-password"
+      classroom = "classroom"
+
+      post :upload_teachers_and_students, params: {
+        school_id: school.id,
+        students: [
+          {
+            name: existing_student.name,
+            email: existing_student.email,
+            teacher_name: existing_teacher.name,
+            teacher_email: existing_teacher.email,
+            classroom: classroom,
+            password: new_password
+          }
+        ],
+        overwritePasswords: true
+      }
+      expect(response.status).to eq 200
+      existing_student.reload
+      expect(existing_student.authenticate(new_password)).to eq(existing_student)
+    end
```

Comment:
This is no huge deal, but we've been shifting toward more `context` blocks for setting up code around `expects`.  Also, whenever I see `reload` I think I want to use `expect ... to change`.  So this whole test would look like this for me:
```ruby
context 'overwritePasswords param`
  subject { post :upload_teachers_and_students, params: params }

  let(:overwrite_passwords) { true }
  let(:new_password) { 'new-password' }
  let(:classroom) { 'classroom' }
  let!(:user) { create(:schools_users, school: school, user: existing_teacher) }
  let(:params) do
    {
      school_id: school.id,
      students: [
        {
          name: existing_student.name,
          email: existing_student.email,
          teacher_name: existing_teacher.name,
          teacher_email: existing_teacher.email,
          classroom: classroom,
          password: new_password
        }
      ],
      overwritePasswords: overwrite_passwords
    }
  end

  it do
    subject
    expect(response.status).to eq 200
  end

  it { expect { subject }.to change { existing_student.authenticate(new_password) }.from(false).to(existing_student) }

  context 'overwritePasswords is false' do
    let(:overwrite_passwords) { false }

    it do
      subject
      expect(response.status).to eq 200
    end

    it { expect { subject }.to not_change { existing_student.authenticate(new_password) }
  end
end
```